Boardfiles/nucleol552zeq/Core/Src/main.cpp

```diff
@@ -131,23 +128,12 @@ int main(void)
   MX_ICACHE_Init();
   MX_TIM3_Init();
   MX_SDMMC1_SD_Init();
-  if (MX_FATFS_Init() != APP_OK) {
-    Error_Handler();
-  }
   /* USER CODE BEGIN 2 */
-  logInit();
-
-  TaskHandle_t hSM = NULL;
-  xTaskCreate(SMTask, "SM", 500U, NULL, osPriorityNormal, &hSM);
 
   /* USER CODE END 2 */
 
-  /* Init scheduler */
-  osKernelInitialize();  /* Call init function for freertos objects (in freertos.c) */
-  MX_FREERTOS_Init();

```

Comment:
I don't think init is needed. We had the init in the repo was because they were auto generated by the ide. I don't see the init was used in the`Mastering Freertos` manual.